extensions/cornerstone/src/init.tsx

```diff
@@ -167,9 +167,26 @@ export default async function init({
           viewportId
         );
 
+        const ohifViewport = cornerstoneViewportService.getViewportInfo(
```

Comment:
Thank you for getting this added - I wonder if the code should migrate somewhere it can be called generically?  Maybe just a helper method taking the viewport, stateSyncService and cornerstoneViewportService?

Reply:
yeah it should eventually get moved